src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function sortList() {
+  const arrPeople = [...list].map(el => {
+    return {
+      'name': el.innerText,
+      'position': el.dataset.position,
+      'salary': el.dataset.salary,
+      'salaryNum': +el.dataset.salary.replace(/[$,]/g, ''),
+      'age': el.dataset.age,
+    };
+  });
+
+  arrPeople.sort((a, b) => b.salaryNum - a.salaryNum);
```

Comment:
I think it's unnecessary to create a new array just to add salaryNum property to it, use salary for sorting